src/napari_imagej/utilities/_module_utils.py

```diff
@@ -471,6 +472,9 @@ def module_execute(
                 or None if this module does not return any layer data.
             """
             try:
+                # Start timing
```

Comment:
This shouldn't be in this PR (although the execution timing is good, and should be in another PR)